samples/pwm/main.cpp

```diff
@@ -14,11 +14,37 @@ int main() {
     IO::init();
 
     IO::PWM& pwm = IO::getPWM<IO::Pin::PC_0>();
+    IO::UART& uart = IO::getUART<IO::Pin::UART_TX, IO::Pin::UART_RX>(9600);
     // 1000000 microseconds (1 second) period
-    pwm.setPeriod(1000000);
+    pwm.setPeriod(100000);
     // 50 % duty cycle
     pwm.setDutyCycle(50);
 
+    uint32_t autoReload = 0;
```

Comment:
You probably want to remove this test code, or if you wanted to keep it, move it to its own test.

Reply:
Nice catch